app/web_framework/cluster_base_handler.py

```diff
@@ -37,6 +38,12 @@ def initialize(self, route_node=None, **kwargs):
         :type route_node: RouteNode
         """
         self._route_node = route_node
+
+        accept_header = self.request.headers.get('Accept')
+        uri = self.request.uri
+        self.api_version = APIVersionHandler.resolve_version(accept_header, uri)
+
+        super().set_header(APIVersionHandler.API_VERSION_HEADER_KEY, self.api_version)
```

Comment:
Is `self.set_header()` a better choice here? Usually `super()` is only needed when calling the superclass from the overriding implementation of the same method in the subclass.